wokspace-loader/src/index.ts

```diff
@@ -14,64 +14,67 @@ const css = require('./style.css');
 import { WebsocketClient } from './json-rpc/websocket-client';
 import { CheJsonRpcMasterApi } from './json-rpc/che-json-rpc-master-api';
 import { getWorkspace, startWorkspace } from './workspace';
+import { Loader } from './loader/loader'
 
-const DEFAULT_WEBSOCKET_CONTEXT = '/api/websocket';
+const WEBSOCKET_CONTEXT = '/api/websocket';
 
-let wsClient = new WebsocketClient();
-let wsUrl = formJsonRpcApiLocation(document.location) + DEFAULT_WEBSOCKET_CONTEXT;
-let masterApi = new CheJsonRpcMasterApi(wsClient);
-let workspaceKey = getWorkspaceKey();
-masterApi.connect(wsUrl).then(() => {
-    getWorkspace(getWorkspaceKey()).then(handleWorkspace).catch(err => {
-        console.error(err);
-    });
-});
+let websocketURL = websocketBaseURL() + WEBSOCKET_CONTEXT;
+let master = new CheJsonRpcMasterApi(new WebsocketClient());
 
-function handleWorkspace(ws: che.IWorkspace) {
-    let machineStatusHandler = (message: any) => {
-        console.log('machine status', message);
-    };
-    let machineOutputHandler = (message: any) => {
-        document.getElementById("output").appendChild(document.createTextNode(message.text));
-    };
-    let workspaceStatusHandler = (message: any) => {
-        console.log('workspace status', message);
-        let workspaceStatus = message.status;
-
-        if (workspaceStatus === 'RUNNING') {
-            openIdeAfterRunning(ws);
-        }
+var loader: Loader = new Loader();
```

Comment:
use `let` for variable, type `Loader` is redundant there

Reply:
Done